From 55ab4e863d811c03e126ef887105e14ffa7994c8 Mon Sep 17 00:00:00 2001
From: "liangtaox.hu" <liangtaox.hu@intel.com>
Date: Tue, 18 Jun 2019 16:01:35 +0800
Subject: [PATCH 2337/2367] media: intel-ipu4: ti960: change gpio pin for
 x3a_ficosa

Message for Open Source:
change ti960 gpio pin

Tracked-On: #PKT-2352
Signed-off-by: huliangtao <liangtaox.hu@intel.com>
---
 drivers/media/platform/intel/ipu4-bxt-p-pdata.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
index 07976a4..3b73100 100644
--- a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
+++ b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
@@ -1325,7 +1325,7 @@ static struct ti960_subdev_info ti960_subdevs_2[] = {
 static struct ti960_pdata ti960_pdata = {
 	.subdev_info = ti960_subdevs,
 	.subdev_num = ARRAY_SIZE(ti960_subdevs),
-	.reset_gpio = GPIO_BASE + 62,
+	.reset_gpio = GPIO_BASE + 63,
 	.suffix = 'a',
 };
 
@@ -1344,7 +1344,7 @@ static struct ipu_isys_subdev_info ti960_sd = {
 static struct ti960_pdata ti960_pdata_2 = {
 	.subdev_info = ti960_subdevs_2,
 	.subdev_num = ARRAY_SIZE(ti960_subdevs_2),
-	.reset_gpio = GPIO_BASE + 69,
+	.reset_gpio = GPIO_BASE + 66,
 	.suffix = 'b',
 };
 
-- 
2.7.4

